less/current/layout/basetopbar.less

```diff
@@ -15,6 +15,16 @@
 .base-top-bar .list-inline li {
   padding: 10px 20px 10px 0px;
 }
+.base-top-bar .top-bar-left {
```

Comment:
styles added to fix the positioning of the search bar since relocating it in the DOM tree broke the existing styling